### ISSUE TITLE
Directory isn't empty problem

### ISSUE THREAD

**Issue by FulcrumA** [remyroy/CDDA-Game-Launcher#165]
Sometimes, when I update the game, setting it to delete previous version after update, launcher reports that the directory isn't empty after downloading said update. The files delete alright, just after a few more seconds, but I can do anything beside retrying update, which downloads the whole thing all over again (even though it's already downloaded as an archive as allowed by a different setting) and also forces me to download sound packs and other mods.

I'd suggest a retry option if the files are not deleted or there's some other error as well as making launcher remember current version/date of downloaded soundpacks/mods so all this is carried over isntead of being discarded when update process will throw an error for some reason.

**Comment by remyroy** [comment]
If you could add the whole error output it would help me locate the issue in the source code.

**Comment by FulcrumA** [comment]
Also, my apologies. Apparently it could be closed, it's a duplicate though of of long-standing bug.
#108

As for output, if the launcher makes a log somewhere - do tell where I should look for it and I'll try to paste it here.

**Comment by remyroy** [comment]
If it only shows in the status bar, there probably isn't any log of it if my memory is correct. Should be all.

**Comment by FulcrumA** [comment]
Yup, only in the status bar. Feel free to do tell if I can help somehow (from the user's perspective as I'd be very cautious about letting me anywhere near the code of anything after all those years) squash this long-standing problem as with frequency of CDDA experimental updates, this gets really bothersome over time.
